### ISSUE TITLE
invalid uBlock filters [logger results] 

### ISSUE THREAD

**Issue by RicoHeat** [uBlockOrigin/uAssets#16666]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

I've been using uBlock's logger on the "ALL" tab after filterlists cache dump to see how many invalid filters I can identify..
I just wanted to report the following invalid results from https://ublockorigin.github.io/uAssets/filters/filters.txt

```
Invalid extended filter in ublock-filters: financerites.*##p:has-text([ad_)
Invalid extended filter in ublock-filters: masterduelmeta.com##:xpath(//*[contains(text(),"adblock")]):upward(3)
Invalid extended filter in ublock-filters: akvi.icu##:xpath(//*[contains(text(),"Disable")])
Invalid extended filter in ublock-filters: sports24.*##:xpath(//*[contains(text(),"Watch HD")])
Invalid extended filter in ublock-filters: washingtonpost.com##:xpath(//*[(text()='AD')]/..)
Invalid extended filter in ublock-filters: ebay.*##.s-item:has-text(Sponsored):xpath(//*[string-length(@aria-labelledby) = 8]):upward(.s-item)
Invalid extended filter in ublock-filters: t-online.de##^script:has-text(toscr')
Invalid extended filter in ublock-filters: ettv.*##:xpath(//*[contains(text(),"VPN")]):upward(2)
Invalid extended filter in ublock-filters: 1337x.*,x1337x.*,1337x.unblock2.xyz,1337x.unblocked.*,1337x.unblockit.*##:xpath(//*[contains(text(),"Hide your IP")]/..)
Invalid extended filter in ublock-filters: spankwire.com##:xpath(//*[contains(text(),"Advertisement")]):upward(1)
Invalid extended filter in ublock-filters: primewire.*##:xpath(//*[contains(text(),"Sponsored")]):upward(2)
```

### Configuration

Standard

**Comment by krystian3w** [comment]
I looks mostly as missed in manually translation: 

https://github.com/gorhill/uBlock/commit/a71b71e4c8a2037fc68970bc8912a76732edaade

It would have to be surrounded by quotation marks and for has-text keep the older version (people reluctant to constantly update uBO).

**Comment by garry-ut99** [comment]
RicoHea :
> * [x]  I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue+sort%3Aupdated-desc) to avoid opening a duplicate issue.

Duplicate of https://github.com/uBlockOrigin/uAssets/issues/1054

<br>

> RicoHea : I just wanted to report the following invalid results

Not an issue, read https://github.com/uBlockOrigin/uAssets/issues/1054#issuecomment-1373917517, there were breaking  changes recently in uBO, and the older filters are intentionally being kept for compatibility with older uBO, each of such filter exists in two copies:
- the older one is compatible with older uBO only (< 1.46) and will be seen as invalid in new uBO (=>1.46)
- and the newer one is compatible with new uBO only (=>1.46) and will be seen as invalid in old uBO (< 1.46)
- example:
https://github.com/uBlockOrigin/uAssets/blob/c89935221d69989345ff0b5696dcb16da8df1eb2/filters/filters.txt#L13513-L13514

<br>

The only invalid filters as for now are the following: https://github.com/uBlockOrigin/uAssets/issues/1054#issuecomment-1410399452, still not fixed, despite a week has passed already, keep in mind these filters will not be sreported as invalid in the Logger, but they are still invalid, they do not work, they are discarded by uBO silently.

**Comment by krystian3w** [comment]
Invalid syntax for latest parser: `/([a-zA-Z0-9#]|\]):(before|after)/`

https://github.com/uBlockOrigin/uAssets/blob/e31f893d0cab4be05a08d60448d9f9de8425c72b/filters/unbreak.txt#L2162

**Comment by krystian3w** [comment]
github.com/uBlockOrigin/uAssets/issues/6769
